### ISSUE TITLE
Kodi Philips Hue error

### ISSUE THREAD

**Issue by AKI-Panda** [mpolednik/script.kodi.hue.ambilight#118]
Stucked Install Ambilight Script.
I dawnloaded Module, Ambilight 1.0, Kodi17.6.
Add-ons - Click box - Install from zip file - script.kodi.hue.ambilight-master
then pop Kodi Philips Hue error. I tryed reboot and re-install. but it does'nt work.
Please Help.

**Comment by evantaur** [comment]
yeah there are some errors on default.py or something, nothing you can do to it unless you know python

**Comment by FrAcTi0N** [comment]
Try to download and install this https://github.com/beenje/script.module.requests/archive/gotham.zip as well. It helped me yesterday.

**Comment by alainwolf** [comment]
Some of names in my Hue setup had special characters in them. I had to rename them, to make the add-on work.
